test/OpenTelemetry.Shims.OpenTracing.Tests/SpanMock.cs

```diff
@@ -43,55 +43,57 @@ public SpanMock(SpanContext spanContext)
 
         public List<KeyValuePair<string, object>> Attributes { get; }
 
-        public SpanContext Context { get; private set; }
+        public override SpanContext Context { get; }
 
-        public bool IsRecording { get; private set; }
+        public override bool IsRecording { get; }
 
-        public Status Status { get; set; }
+        public override Status Status { set => this.status = value; }
```

Comment:
Same as in `SpanSdk` - we could use `new` instead of override to keep the inline property accessors.